test/simplehashregistrar_test.js

```diff
@@ -88,9 +88,12 @@ describe('SimpleHashRegistrar', function() {
 
 	it('starts auctions', function(done) {
 		async.series([
-			function(done) {
+			function(done) { 
+				advanceTime(days(100), done); 
```

Comment:
No, I mean the 'launchLength' JS variable you define later in a test. If that's public, then you could use that everywhere you're currently using `advanceTime(days(90))` or `advanceTime(days(100))`, which would make your intent clearer.